pom.xml

```diff
@@ -2,7 +2,7 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.1.0</modelVersion>
```

Comment:
This is wrong, the last version was 2.9.0, you shouldn't update this version.

Reply:
You need to update this file instead https://github.com/onfido/onfido-java/blob/master/onfido-java/pom.xml#L7